website/docs/cli/cloud/settings.html.md

```diff
@@ -0,0 +1,103 @@
+  [`terraform login`](/docs/cli/commands/login.html) command.
+- Add a `cloud` block to the directory's Terraform configuration, to specify
+  which organization and workspace(s) to use.
+- Optionally, use a `.terraformignore` file to specify files that shouldn't be
+  uploaded with the Terraform configuration when running plans and applies.
+
+After adding or changing a `cloud` block, you must run `terraform init`.
+
+## The `cloud` Block
+
+The `cloud` block is a nested block within the top-level `terraform` settings
+block. It specifies which Terraform Cloud workspaces to use for the current
+working directory.
+
+```hcl
+terraform {
+  cloud {
+    organization = "my-org"
+    hostname = "app.terraform.io" # Optional; defaults to app.terraform.io
+
+    workspaces {
+      tags = ["networking", "source:cli"]
+    }
+  }
+}
+```
+
+The `cloud` block has some special restrictions:
+
+- A configuration can only provide one `cloud` block.
+- A `cloud` block cannot be used with [state backends](/docs/language/settings/backends/index.html).
+  A configuration can use one or the other, but not both.
+- A cloud block cannot refer to named values (like input variables, locals, or
+  data source attributes).
+
+The `cloud` block only affects Terraform CLI's behavior. When Terraform Cloud uses a configuration
+that contains a cloud block - for example, when a workspace is configured to use a VCS provider
+directly - it ignores the block and behaves according to its own workspace settings.
+
+### Arguments
+
+The `cloud` block supports the following configuration arguments:
+
+* `organization` - (Required) The name of the organization containing the
+  workspace(s) the current configuration should use.
+
+* `workspaces` - (Required) A nested block that specifies which remote Terraform Cloud workspaces to
+use for the current configuration. The `workspaces` block must contain **exactly one** of the
+following arguments, each denoting a strategy for how workspaces should be mapped:
+
+    * `tags` - (Optional) A set of Terraform Cloud workspace tags. You will be able to use
+      this working directory with any workspaces that have all of the specified tags,
+      and can use [the `terraform workspace` commands](/docs/cli/workspaces/index.html)
+      to switch between them or create new workspaces; new workspaces will automatically have
+      the specified tags. This option conflicts with `name`.
+
+    * `name` - (Optional) The name of a single Terraform Cloud workspace. You will
+      only be able to use the workspace specified in the configuration with this working
+      directory, and cannot manage workspaces from the CLI (e.g. `terraform workspace select` or
+      `terraform workspace new`). This option conflicts with `tags`.
+
+* `hostname` - (Optional) The hostname of a Terraform Enterprise installation, if using Terraform
+  Enterprise. Defaults to Terraform Cloud (app.terraform.io).
```

Comment:
```suggestion
* `hostname` - (Optional) The hostname of a Terraform Enterprise installation. Defaults to Terraform Cloud (app.terraform.io).
```

Reply:
This is a little bit weird. Lets leave it as-is

Reply:
```suggestion
- `hostname` - (Optional) The hostname of a Terraform Enterprise installation, if using Terraform
  Enterprise. Defaults to Terraform Cloud (app.terraform.io).
```